gradle.properties

```diff
@@ -12,5 +12,5 @@ fileProviderPath="raw_data/"
 # - app store: me.clarius.clarius
 # - pilot: me.clarius.clarius.release
 # - dev: me.clarius.clarius.develop.developer.debug
-clariusPackageName="me.clarius.clarius"
```

Comment:
We should keep the original package name because the normal use case is to connect to the currently released app, most partners do not use development version of the Clarius App (also update the change log as appropriate).